rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1259,6 +1260,36 @@ def test_latest_upgrade_adds_remove_tables(user_data_dir):
     assert new_views == set()
 
 
+def test_steps_counted_properly_in_upgrades(user_data_dir):
```

Comment:
Guess this is also fine. But you got one more test running through all the upgrades while you could have done it in each of the already existing tests.

I think this is fine though, since I guess it does not take too long and is easier to write this test rather than repeating same code in all existing tests.